src/main/kotlin/racing/domain/Car.kt

```diff
@@ -0,0 +1,29 @@
+package racing.domain
+
+private const val MAXIMUM_NAME_LENGTH = 5
+private const val DEFAULT_POSITION = 1
+private const val MOVABLE_MINIMUM = 4
+private const val MOVING_DISTANCE = 1
+
+data class Car(val name: String, private var _position: Int = DEFAULT_POSITION) {
```

Comment:
```suggestion
data class Car(
    val name: String,
    private var _position: Int = DEFAULT_POSITION
) {
```
(반영하지 않으셔도 됩니다) `CarRacingResult`에서 해주신 이런 포맷이 좀 더 가독성이 있더라구요.